plugins/foreman_kubevirt/0.x/index.md

```diff
@@ -0,0 +1,331 @@
+---
+layout: plugin
+pluginname: foreman_kubevirt
+title: Foreman KubeVirt 0.x Manual
+version: 0.x
+---
+
+# 1. {{ page.title }}
+
+KubeVirt is a virtualization add-on to Kubernetes. It allows you to run a VM inside of a pod/container and have that VM be managed by Kubernetes.
+KubeVirt technology addresses the needs of development teams that have adopted or want to adopt Kubernetes but possess existing Virtual Machine-based workloads that cannot be easily containerized.
+
+For more information see ["KubeVirt"](https://kubevirt.io/).
+
+You can find the latest version of the foreman-kubevirt plugin [on Github](https://github.com/masayag/foreman_kubevirt).
+
+## Compatibility
+
+<table class="table table-bordered table-condensed">
+  <thead>
+    <tr>
+      <th align="right">Foreman</th>
+      <th align="right">Plugin</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td align="right">&gt;= 1.7</td>
```

Comment:
does it support 1.7+ ? i assume you didn't test it with 1.7? I guess its safer to use something more recent?

Reply:
Updated to 1.21 which is the actual version we've started working on and verified against.